### PR TITLE
Login modal and form components

### PR DESCRIPTION
**Pull request by mattmcfad**
Connected to rangle/rangle-starter#17

### CODE REVIEW

**Comment by SethDavenport** [comment]
Look good (apart from the linter error).  I'd like one of our ng2 gurus to take a look as well (since I'm better at React).

@e-schultz @bennett000 @bertrandk any takers?

File: src/components/form/form.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Component } from 'angular2/core';
+
+@Component({
+  selector: 'form',
```
Comment (mattmcfad) [inline src/components/form/form.ts]: I'm wondering if we should rename this selector? Will it cause problems with the native `<form>`?

Comment (SethDavenport) [inline src/components/form/form.ts]: yeah - I wonder if we should be namespacing our components generally (in react or angular), e.g. rio-form, RioForm or similar?

@bertrandk thoughts?

Comment (bennett000) [inline src/components/form/form.ts]: yes this selector should definitely be renamed :+1:

Comment (bennett000) [inline src/components/form/form.ts]: On the subject of namespacing I _was_ a big fan even though it adds visual clutter.

Comment (SethDavenport) [inline src/components/form/form.ts]: We encouraged namespacing in angular 1 so I'm inclined to continue namespacing in angular 2 - I don't think the underlying reasoning has changed. Actually it now seems even more important with the emphasis on components.

However I'd like it to be consistent across the seeds.

What I suggest is rename this selector for now to something that doesn't clash with HTML - let's say `custom-form` and I'll file a separate issue to tackle namespacing holistically across all four seeds.

**Comment by SethDavenport** [comment]
So rename the selectors, squash, and :shipit:

**Comment by SethDavenport** [comment]
Logged issue rangle/rangle-starter#52

**Pull request merged**